cmd/snap/cmd_get.go

```diff
@@ -77,6 +82,63 @@ func init() {
 		})
 }
 
+type ConfigValue struct {
+	Path  string
+	Value interface{}
+}
+
+type byConfigPath []ConfigValue
+
+func (s byConfigPath) Len() int      { return len(s) }
+func (s byConfigPath) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
+func (s byConfigPath) Less(i, j int) bool {
+	other := s[j].Path
+	for k, c := range s[i].Path {
+		if len(other) <= k {
+			return false
```

Comment:
this code path doesn't seem exercized by the tests

Reply:
Ok, added an extra input to the test to trigger this.